index.bs

```diff
@@ -1409,6 +1361,70 @@ you can extend the {{WindowOrWorkerGlobalScope}} [=interface mixin=] using a [=p
     };
 </pre>
 
+
+<h3 id="idl-callback-interfaces">Callback interfaces</h3>
+
+A <dfn id="dfn-callback-interface" export>callback interface</dfn> is a [=definition=] matching
+<emu-t>callback</emu-t> <emu-t>interface</emu-t> <emu-nt><a href="#prod-InterfaceRest">InterfaceRest</a></emu-nt>.
+It can be implemented by any object, as described in [[#idl-objects]].
```

Comment:
Can we note somewhere around here (i.e. early) that the name "callback interfaces" is a legacy holdover?